### PR TITLE
docs: add how to add X-Requested-With header with htmx

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
- add how to add X-Requested-With header with htmx

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: user_guide_src/source/general/ajax.rst [hunk 1]
```diff
@@ -52,3 +52,14 @@ React
 .. code-block:: javascript
 
     axios.get("your url", {headers: {'Content-Type': 'application/json'}})
+
+htmx
+=====
```
Comment (datamweb) [inline user_guide_src/source/general/ajax.rst:57]:
```suggestion
====
```

**Review by datamweb — CHANGES_REQUESTED** [review]

File: user_guide_src/source/general/ajax.rst (hunk 1 above)
Comment (kenjis) [reply user_guide_src/source/general/ajax.rst]: Oh, thanks!

**Review by MGatner — APPROVED** [review]
Good by me! Maybe give @lonnieezell a day then go for it - we can always pull it later.

**Review by lonnieezell — APPROVED** [review]
I like this addition. We already have the other big ones, and I'd love to push htmx more, honestly.

**Review by michalsn — APPROVED** [review]

**Comment by jozefrebjak** [comment]
@kenjis Nice!

**Pull request merged**